spec/unit/block_score/base_spec.rb

```diff
@@ -14,22 +14,90 @@
       subject(:candidate) { create(:candidate, name_first: 'John') }
       before do
         candidate.name_first = 'Jane'
-        candidate.save
+        result = candidate.save
+        expect(result).to be true
```

Comment:
I don't think there's any reason to assign this to the `result` lvar. This is a [synthetic variable](http://www.perl.com/pub/2000/11/repair3x.html#synthetic variables) and not necessary to solve the problem.